src/main/scala/com/raphtory/core/components/partition/BatchWriter.scala

```diff
@@ -92,10 +95,12 @@ class BatchWriter[T: ClassTag](
     logger.trace(s"Partition $partitionID: Received EdgeDelete message '$update'.")
     storage.timings(update.updateTime)
     storage.removeEdge(update.updateTime, update.srcId, update.dstId)
+    PartitionTelemetry.batchWriterEdgeDeletions.inc()
```

Comment:
This should be addition not deletion, also probably should be remote edge additions (a new number to increment).